alembic/versions/104d4a614d95_add_performancestat.py

```diff
@@ -10,27 +10,30 @@
 
 
 # revision identifiers, used by Alembic.
-revision = '104d4a614d95'
-down_revision = 'b2dc41850120'
+revision = "104d4a614d95"
```

Comment:
@Mustaballer can you please modify the configuration such that files in this directory are ignored? They are auto-generated and can stay as-is.

Reply:
Yeah sure will do :)